gateway/src/apicast/policy/rate_limit/README.md

```diff
@@ -15,8 +15,8 @@ The purpose of this policy is to control the rate of traffic sent to the
 upstream API. There are several kinds of limits that can be defined. Here are
 some examples:
 
-- The client with address 127.0.0.1 can access the specific endpoint
-`/something` up to 100 times per minute.
+- The client with address 127.0.0.1 can access the specific API
```

Comment:
I think that both things should be possible using this policy. You can limit a whole API/service but you can also limit specific endpoints/paths as it said in the deleted part of the diff.